src/templates/csr/fabric-expandable/index.svelte

```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
+	import type { GAMVariable } from '$lib/gam';
+	import { post } from '$lib/messenger';
+	import Fabric from '$templates/components/Fabric.svelte';
+	import ArrowDown from '$templates/components/icons/ArrowDown.svelte';
+	import CrossIcon from '$templates/components/icons/CrossIcon.svelte';
+
+	export let Trackingpixel: GAMVariable;
+	export let Researchpixel: GAMVariable;
+	export let Viewabilitypixel: GAMVariable;
+
+	export let ScrollType: GAMVariable<'parallax' | 'none' | 'fixed'>;
+
+	export let Slide1BackgroundColour: GAMVariable;
+	export let Slide1BackgroundImage: GAMVariable;
+	export let Slide1BackgroundRepeat: GAMVariable;
+	export let Slide1BackgroundPosition: GAMVariable;
+	export let Slide1MobileBackgroundImage: GAMVariable;
+	export let Slide1MobileBackgroundRepeat: GAMVariable;
+	export let Slide1MobileBackgroundPosition: GAMVariable;
+	export let Slide1Layer1BackgroundImage: GAMVariable;
+	export let Slide1Layer1BackgroundPosition: GAMVariable;
+	export let Slide1Layer1MobileBackgroundImage: GAMVariable;
+	export let Slide1Layer1MobileBackgroundPosition: GAMVariable;
+	export let Slide1Layer2BackgroundImage: GAMVariable;
+	export let Slide1Layer2BackgroundPosition: GAMVariable;
+	export let Slide1Layer2MobileBackgroundImage: GAMVariable;
+	export let Slide1Layer2MobileBackgroundPosition: GAMVariable;
+	export let Slide1Layer3BackgroundImage: GAMVariable;
+	export let Slide1Layer3BackgroundPosition: GAMVariable;
+	export let Slide1Layer3MobileBackgroundImage: GAMVariable;
+	export let Slide1Layer3MobileBackgroundPosition: GAMVariable;
+
+	export let Slide2BackgroundColour: GAMVariable;
+	export let Slide2BackgroundImage: GAMVariable;
+	export let Slide2BackgroundRepeat: GAMVariable;
+	export let Slide2BackgroundPosition: GAMVariable;
+	export let Slide2MobileBackgroundImage: GAMVariable;
+	export let Slide2MobileBackgroundRepeat: GAMVariable;
+	export let Slide2MobileBackgroundPosition: GAMVariable;
+	export let Slide2Layer1BackgroundImage: GAMVariable;
+	export let Slide2Layer1BackgroundPosition: GAMVariable;
+	export let Slide2Layer1MobileBackgroundImage: GAMVariable;
+	export let Slide2Layer1MobileBackgroundPosition: GAMVariable;
+	export let Slide2Layer2BackgroundImage: GAMVariable;
+	export let Slide2Layer2BackgroundPosition: GAMVariable;
+	export let Slide2Layer2MobileBackgroundImage: GAMVariable;
+	export let Slide2Layer2MobileBackgroundPosition: GAMVariable;
+	export let Slide2Layer3BackgroundImage: GAMVariable;
+	export let Slide2Layer3BackgroundPosition: GAMVariable;
+	export let Slide2Layer3MobileBackgroundImage: GAMVariable;
+	export let Slide2Layer3MobileBackgroundPosition: GAMVariable;
+
+	post({ type: 'resize', value: { height: 250 } });
+
+	let expanded = false;
+
+	function toggleExpanded() {
+		expanded = !expanded;
+		height = expanded ? 500 : 250;
+		post({ type: 'resize', value: { height } });
+	}
+
+	let height: number = -1;
+</script>
+
+<aside bind:clientHeight={height}>
```

Comment:
can make this a bit tidier with a reactive statement `$:`
```suggestion
	let expanded = false;

	const toggleExpanded = () => (expanded = !expanded);

	$: post({ type: 'resize', value: { height: expanded ? 500 : 250 } });
</script>

<aside>
```

Reply:
Much more succinct - nice Svelte feature